src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,84 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  buttonSwitch: Todo | null;
```

Comment:
also about naming
It's not clear what is `buttonSwitch` here. It has Todo type (or null), so actually it's todo. So you should name this prop accordingly - `selectedTodo`. `selected` describes what kind of todo it is

naming is very important part of clear and easy to read code 🙂